htcrystalball/main.py

```diff
@@ -1,12 +1,17 @@
 """Entry point for HTCrystalBall examination and collection."""
 
-from . import __version__, collect, examine
-from .utils import validate_storage_size, validate_duration
 import argparse
 import sys
+from . import __version__, collect, examine
+from .utils import validate_storage_size, validate_duration
 
 
 def main():
+    """
+    Defines the command line parser and argument properties
+    Returns:
+
+    """
```

Comment:
Line 12 (`Returns:`) can be removed as the function returns nothing. You could change line 9 to `def main() -> None:` to indicate this.